sdk/cosmosdb/cosmos/src/client/Item/Item.ts

```diff
@@ -148,8 +154,9 @@ export class Item {
       throw err;
     }
 
-    const path = getPathFromLink(this.url);
-    const id = getIdFromLink(this.url);
+    const resourceUri: string = this.url;
```

Comment:
can't we reuse the same constant everywhere which is defined above?

Reply:
different methods - and possibly uri values - the const is scoped to each of the methods